packages/ibm-products/src/components/WebTerminal/WebTerminal.tsx

```diff
@@ -279,4 +287,9 @@ WebTerminal.propTypes = {
    * Optionally pass if the web terminal should be open by default
    */
   isInitiallyOpen: PropTypes.bool,
+
+  /**
+   * Specifies aria label for Web terminal
+   */
+  webTerminalAriaLabel: PropTypes.string.isRequired,
```

Comment:
I think to minimize impact to developer experience, we should set a fallback in [L185](https://github.com/carbon-design-system/ibm-products/pull/5171/files#diff-92f66e28307c3809b4bf6bd1fd6ac70644ed33b32846e25267fd36bb91d0f6e4R185) or set a default prop for the `webTerminalAriaLabel`.

Requiring this could cause errors for anyone using this component.